src/Libraries/DynamoArduino/Arduino.csproj

```diff
@@ -20,6 +23,9 @@
     <Reference Include="PresentationFramework" />
     <Reference Include="PresentationCore" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Windows.Compatibility" Version="6.0.7" Condition=" '$(TargetFramework)' == 'net6.0-windows' "/>
```

Comment:
this appears to bring in a million things:
https://www.nuget.org/packages/Microsoft.Windows.Compatibility

1. why do we need it?
2. I vote to remove this project.

Reply:
For the last couple years I never see Dynamo actually use this project right?

Reply:
Yes, the package is needed for serial port support. Not a problem anymore. I will remove the project instead.